### PR TITLE
[CLB-20] Implement Login Page

### PR DESCRIPTION
**Pull request by Mbotunjac**
Implemented Login page, and also fixed some colors in themes.modal.css. 
(Had a typo in success-color, added yellow-color, added blue-hover-color and fixed in body: { color: --text-primary-color }; so you don't have to write white every time now)

P.S. That button color in Login page is ugly as hell so I do accept any designer suggestions :)

### CODE REVIEW

File: src/App.tsx
Comment (kimnovak) [inline src/App.tsx]: Since you implemented the whole page it can be `import LoginPage` and update the file name as well

File: src/App.tsx
Comment (kimnovak) [inline src/App.tsx]:
```suggestion
                <Route path="/login" element={<LoginPage />} />
```

File: src/components/Login/Login.module.css
Comment (kimnovak) [inline src/components/Login/Login.module.css]: Variables are added to the root so I'm wondering if we need this import?

File: src/components/Login/Login.module.css
Comment (kimnovak) [inline src/components/Login/Login.module.css]: Defining margins and padding in %s can be buggy, my suggestion is to convert to pixels

File: src/components/Login/Login.module.css
Comment (kimnovak) [inline src/components/Login/Login.module.css:38]: Probably needs this too
```suggestion
    cursor: pointer;
}
```

File: src/components/Login/Login.module.css
Comment (kimnovak) [inline src/components/Login/Login.module.css]: For consistency throughout this file 
```suggestion
.login {
```
You will need to update the name everywhere it's used

File: src/components/Login/index.tsx
Comment (kimnovak) [inline src/components/Login/index.tsx]: I think this should be enough, because empty string and undefined will be falsy
```suggestion
        return email && password;
```

File: src/theme.module.css
Comment (kimnovak) [inline src/theme.module.css:16]: Have you checked if the "succses-color" is used anywhere else and if it needs to be updated there too?

**Review by kimnovak — CHANGES_REQUESTED** [review]

File: src/App.tsx
Comment (Mbotunjac) [reply src/App.tsx]: Sounds good, and easy to read. will make a change

File: src/components/Login/index.tsx
Comment (Mbotunjac) [reply src/components/Login/index.tsx]: wanted to make sure, but when I think about it, this is probably better.  Will make a change.

File: src/components/Login/Login.module.css
Comment (Mbotunjac) [reply src/components/Login/Login.module.css:38]: yes, didnt think of it, great idea

File: src/components/Login/Login.module.css
Comment (Mbotunjac) [reply src/components/Login/Login.module.css]: great suggestion, will make a change

File: src/components/Login/Login.module.css
Comment (Mbotunjac) [reply src/components/Login/Login.module.css]: Didnt think on it, good point!

File: src/components/Login/Login.module.css
Comment (Mbotunjac) [reply src/components/Login/Login.module.css]: yes, makes more sense

File: src/theme.module.css
Comment (Mbotunjac) [reply src/theme.module.css:16]: yes, I did, no use for it yet, only "green-color" has been used (witch is the same one)

**Comment by kimnovak** [comment]
Looks good to me, after someone else reviews it too, it can be merged if no changes get requested

**Comment by Zola987** [comment]
Outstanding, i couldn't find any mistakes.

**Review by bojanab987 — APPROVED** [review]
Looks good!

Only when rendering looks like some input alignments are not as should be.
I believe this is because of earlier done css modules where styling is defined with input tag, so this interferes with yours styling. 
image

**Comment by Mbotunjac** [comment]
> Looks good!
> 
> Only when rendering looks like some input alignments are not as should be. I believe this is because of earlier done css modules where styling is defined with input tag, so this interferes with yours styling. image

Great point, it is. I did saw that but actually liked the way it looks, kinda more "lively"  :D

**Pull request merged**
